runner/query_run.go

```diff
@@ -0,0 +1,131 @@
+package runner
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pganalyze/collector/input/postgres"
+	"github.com/pganalyze/collector/output/pganalyze_collector"
+	"github.com/pganalyze/collector/state"
+	"github.com/pganalyze/collector/util"
+)
+
+func SetupQueryRunnerForAllServers(ctx context.Context, servers []*state.Server, collectionOpts state.CollectionOpts, logger *util.Logger) {
+	if collectionOpts.ForceEmptyGrant {
+		return
+	}
+	for idx := range servers {
+		go func(server *state.Server) {
+			logger = logger.WithPrefixAndRememberErrors(server.Config.SectionName)
+			cleanupInterval := time.NewTicker(5 * time.Minute)
+			for {
+				select {
+				case <-ctx.Done():
+					return
+				case <-cleanupInterval.C:
+					cleanup(server)
+				default:
+					if server.Config.EnableQueryRunner {
```

Comment:
Do you place this if statement here instead of using this to early return because you want to run cleanup? Curious why you're putting here.